lighthouse-core/test/audits/service-worker-test.js

```diff
@@ -165,10 +162,9 @@ describe('Offline: service worker audit', () => {
     const manifest = {start_url: startUrl};
 
     const output = ServiceWorker.audit(createArtifacts(swOpts, finalUrl, manifest));
-    expect(output).toMatchObject({
-      score: 0,
-      explanation: expect.stringMatching(new RegExp(`start_url.*${startUrl}.*${scopeURL}`)),
-    });
+    assert.strictEqual(output.score, 0);
+    expect(output.explanation).toBeDisplayString('This page is controlled by a service worker, ' +
```

Comment:
could leave the regex checks here FWIW `toBeDisplayString` should support it

Reply:
Good to know.  Iirc I actually liked this better b/c I changed the string at some point and it didn't throw and I was like 😕 Most of the display string tests are like this.  I have no strong opinions though.

Reply:
same with these changes. It's not good that something wasn't caught, so maybe they needed tweaking, but if the chosen regex checks that the proper information is coming through, the exact wording isn't that important (we have other tests that check that strings in UIStrings make it through to audit explanations/warnings/etc)

There's always a precision tradeoff, but we generally don't want to require a change to a bunch of tests if one day we want to capitalize `Service Worker` or something...that's not what these tests are meant to test.